v0/sources/mailmodo/mapping.json

```
@@ -0,0 +1,14 @@
+[
+  {
+    "sourceKeys": [
+      "triggerData.email",
+      "recipientData.email",
+      "recipientEmail"
+    ],
+    "destKeys": "context.traits.email"
+  },
+  {
+    "sourceKeys": "recipientData",
+    "destKeys": "context.traits.recipientData"
```

```suggestion
    "destKeys": "context.traits"
```
if both `recipientEmail` and `recipientData` are present then remove one email.

---

Done